boardswarm-cli/src/ui.rs

```
@@ -223,23 +259,35 @@ pub async fn run_ui(
                 async move {
                     match i {
                         Input::PowerOn => {
-                            device.change_mode("on").await.unwrap();
+                            let _ = device.change_mode("on").await;
```

while just unwrapping isn't great; hiding the failure isn't good either :)